src/components/ui/PriceRange/price-range.module.scss

```
@@ -6,71 +6,103 @@
   // --------------------------------------------------------
 
   // Default properties
-  --fs-price-range-slider-height: .5rem;
-  --fs-price-range-slider-border-radius: var(--fs-border-radius);
+  --fs-price-range-height                   : var(--fs-spacing-2);
+  --fs-price-range-border-radius            : var(--fs-border-radius);
+  --fs-price-range-margin-bottom            : var(--fs-spacing-3);
+
+  --fs-price-range-transition-function      : var(--fs-transition-function);
+  --fs-price-range-transition-property      : var(--fs-transition-property);
+  --fs-price-range-transition-timing        : var(--fs-transition-timing);
+
+  --fs-price-range-value-label-bottom       : var(--fs-spacing-4);
+  --fs-price-range-value-label-bkg-color    : var(--fs-color-body-bkg);
 
   // Slider
-  --fs-price-range-slider-bkg-color: var(--fs-color-primary-bkg-light);
-  --fs-price-range-slider-range-bkg-color: var(--fs-color-primary-bkg);
+  --fs-price-range-slider-bkg-color         : var(--fs-color-neutral-bkg);
+  --fs-price-range-selection-bkg-color      : var(--fs-color-primary-bkg-light);
```

Maybe slider-selection gets more semantic?
```suggestion
  --fs-price-range-slider-selection-bkg-color      : var(--fs-color-primary-bkg-light);
```